area/area.go

```diff
@@ -135,3 +141,57 @@ func (m *GormAreaRepository) ListChildren(ctx context.Context, parentArea *Area)
 	}
 	return objs, nil
 }
+
+// Root fetches the Root Areas inside a space.
+func (m *GormAreaRepository) Root(ctx context.Context, spaceID uuid.UUID) (*Area, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "Area", "root"}, time.Now())
+	var rootArea []Area
+
+	parentPathOfRootArea := path.Path{}
+	rootArea, err := m.Query(FilterBySpaceID(spaceID), FilterByPath(parentPathOfRootArea))
```

Comment:
Just a comment  : no need to change the code.
But just mentioning : It could have been a simple to read -
`m.db.Where("space_id = ? and path = ?", spaceID, "").First(&rootArea)`

At the same time, I also see the strength `Query` function adds to the repository. It is good.